pandas/computation/ops.py

```diff
@@ -166,6 +166,11 @@ def _resolve_name(self):
     def name(self):
         return self.value
 
+    def __unicode__(self):
+        # in python 2 str() of float
+        # can truncate shorter than repr()
+        return repr(self.name)
```

Comment:
I bet this is a similar problem when selecting from pytables, can you add a test for that?